app/lib/primer/classify/functional_colors.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+module Primer
+  class Classify
+    # https://primer-css-git-mkt-color-modes-docs-primer.vercel.app/css/support/v16-migration
+    class FunctionalColors
```

Comment:
@BlakeWilliams may I get some help with allocations added here? I see the test failing 😬